client/.env.SAMPLE

```diff
@@ -1,3 +0,0 @@
-This is a sample version of .env file. Provide values before running the app.
-
-SKIP_PREFLIGHT_CHECK=
```

Comment:
don't remove. but add keys so devs are on the same page about keys required.
add REACT_APP_MAPBOX_TOKEN here.